ansible/playbooks/AdoptOpenJDK_Unix_Playbook/trss.yml

```diff
@@ -0,0 +1,157 @@
+---
+############################################
+# AdoptOpenJDK Ansible TRSS Playbook       #
+# Currently supports Ubuntu 18 on x64 only #
+############################################
+- hosts: all
+  user: root
+  become: yes
+  tasks:
+    - block:
+        - name: Load AdoptOpenJDKs variable file
+          include_vars: group_vars/all/adoptopenjdk_variables.yml
+
+        - name: Set hostname to trss.adoptopenjdk.net
+          hostname:
+            name: trss.adoptopenjdk.net
+          tags: hostname
+
+        - name: OS update -- apt-get upgrade
+          apt: upgrade=safe update_cache=yes
+          tags: patch_update
+
+        - name: Add Node.js v12.x
+          raw: "curl -sL https://deb.nodesource.com/setup_12.x | sudo -E bash -"
```

Comment:
I'm not comfortable with using that way to install Node.
If we want it regularly patched with security releases via `apt-get upgrade` then I'd prefer us to use the ansible operations which add that repository manually instead of using that shell script from an unofficial source.
If we don't want it regularly updated, then grabbing it directly from the nodejs.org site would be preferable

Reply:
Those installation instructions were found from a link on the nodejs.org website, https://nodejs.org/en/download/package-manager/#debian-and-ubuntu-based-linux-distributions-enterprise-linux-fedora-and-snap-packages

I cant seem to find a way to add the repository using 'ansibley', but an alternative could be to download and configure/make from source code. Would this be viable?

Reply:
No reason to compile from source, you can just pull down a tarball from e.g. https://nodejs.org/dist/v12.16.1/node-v12.16.1-linux-x64.tar.xz

Although something like https://github.com/AdoptOpenJDK/openjdk-infrastructure/blob/master/ansible/playbooks/AdoptOpenJDK_Unix_Playbook/roles/Common/tasks/Ubuntu.yml#L37 is an example of adding a third party apt repository